velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1862,3 +1862,55 @@ TEST_F(StringFunctionsTest, varbinaryLength) {
   auto result = evaluate("length(c0)", makeRowVector({vector}));
   test::assertEqualVectors(expected, result);
 }
+
+TEST_F(StringFunctionsTest, normalize) {
+  const auto normalizeWithoutForm = [&](std::optional<std::string> string) {
+    return evaluateOnce<std::string>("normalize(c0)", string);
+  };
+
+  const auto normalizeWithForm = [&](std::optional<std::string> string,
+                                     const std::string& form) {
+    return evaluateOnce<std::string>(
+        fmt::format("normalize(c0, '{}')", form), string);
+  };
+
+  std::string inValidTestString = "\xEF\xBE\x8";
+  std::string testStringOne =
```

Comment:
Move these definitions closer to where they are used.

Reply:
Done.